src/index.js

```diff
@@ -0,0 +1,85 @@
+/*eslint-disable*/
+import _ from 'lodash'; /*eslint-disable*/
```

Comment:
Kindly remove this as its an unused in this project.

Reply:
Hello @cosywasswa . Thank you for your kind insights. However, I used anonymous function in my case. When the submitButton is clicked, the anonymous function will be executed. In this case, it is assumed that the function Functionality.newTask() is called inside the anonymous function, indicating that it handles the creation of a new task. For that I got an error "unnamed function".